composer.json

```diff
@@ -29,10 +29,10 @@
       }
     },
     "require-dev": {
-        "phpunit/phpunit": "^5"
+        "phpunit/phpunit": "^8.0"
```

Comment:
Could this be v9?

```suggestion
        "phpunit/phpunit": "^9.0"
```

Reply:
It'd be good to add the `.phpunit.result.cache` file to gitignore as well

Reply:
Could be - Wasn't sure if you wanted to support V8 as it's not EOL yet

Reply:
gitignore updated - 74bbf9c2ac2569043871da6aa82b63ac21987a25

Reply:
> Could be - Wasn't sure if you wanted to support V8 as it's not EOL yet

This is just `require-dev` so it only affects the library's test suite 🙂